operator/v1/types_ingress.go

```diff
@@ -1525,16 +1525,17 @@ type IngressControllerTuningOptions struct {
 	// A zero value for reloadInterval tells the IngressController to choose the default,
 	// which is currently 5s and subject to change without notice.
 	//
-	// This field expects an unsigned duration string of decimal numbers, each with optional
-	// fraction and a unit suffix, e.g. "100s", "1m30s". Valid time units are "s" and "m".
+	// Expects an unsigned duration string of decimal numbers, each with optional
+	// fraction and a unit suffix, eg "300ms", "1.5h" or "2h45m".
```

Comment:
Eeeeeurgh...  I kinda prefer "e.g.".  The IBM Style Guide actually recommends spelling out "for example".  I suppose it isn't all that important though.  Why drop "This field"?

Reply:
I just copied and pasted from another "duration string" field, I'll add it back in this field for consistency and add the periods.